code/game/mecha/mecha.dm

```diff
@@ -841,6 +842,8 @@
 			user << "<span class='warning'>You can't enter the exosuit with other creatures attached to you!</span>"
 		else
 			moved_inside(user)
+			if(user.fov)
+				user.fov.alpha = 0 //snowflake for vision cone

```

Comment:
I did that, it updated the sprite but not the actual list cone. So you could still see everywhere around you.